es6-arrow-functions/main.js

```diff
@@ -0,0 +1,45 @@
+const $jokeContainer = document.querySelector('#joke-container');
+const $flashImage = document.querySelector('#flash-image');
+const $jokeForm = document.querySelector('#joke-form');
+
+const jokester = {
+  tellJoke: function (setup, punchline) {
+    $jokeForm.classList.add('d-none');
+    $jokeContainer.classList.replace('d-none', 'd-flex');
+    const $introStatement = this.renderJokePhrase('Hey Flash...');
+    this.appendJokePhrase($introStatement);
+    setTimeout(() => {
+      const $jokeSetup = this.renderJokePhrase(setup);
+      this.appendJokePhrase($jokeSetup);
+      setTimeout(() => {
+        const $jokePunchline = this.renderJokePhrase(punchline);
+        this.appendJokePhrase($jokePunchline);
+        flash.laugh();
+      }, 2000);
+    }, 2000);
+  },
+  renderJokePhrase: phrase => {
```

Comment:
All object methods have to be standard functions or you'll eventually run into problems that aren't exhibited in this exercise. So change this one back to use `function`. However, with es6 classes you probably won't ever run into this again, so don't worry about it!

Check the one below, too.